docs/compilers/Dynamic Analysis/MetadataFormat.md

```diff
@@ -0,0 +1,48 @@
+#### <a name="MethodSpansBlob"></a>Method Spans Blob
+Span is a quadruple of integers and a document reference:
+
+* Start Line
+* Start Column
+* End Line
+* End Column
+* Document
+
+The values of a span must satisfy the following constraints
+
+* Start Line is within range [0, 0x20000000).
+* End Line is within range [0, 0x20000000).
+* Start Column is within range [0, 0x10000)
+* End Column is within range [0, 0x10000)
+* End Line is greater or equal to Start Line.
+* If Start Line is equal to End Line then End Column is greater than Start Column.
+
+_Method Spans Blob_ has the following structure:
+
+    Blob ::= header span-record (span-record | document-record)*
```

Comment:
Oh, is the idea that a document record precedes a list of span records from that document?